src/App.tsx

```diff
@@ -1,61 +1,103 @@
-import React from 'react';
+import { useState, FC } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { getReorderedGoods } from './functions';
+import { SortType } from './types';
+import goodsFromServer from './goodsFromServer';
+
+export const App: FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+
+  const sortAlphabetically = () => {
+    if (sortType !== SortType.ALPHABET) {
+      setSortType(SortType.ALPHABET);
+    }
+  };
+
+  const sortByLength = () => {
+    if (sortType !== SortType.LENGTH) {
+      setSortType(SortType.LENGTH);
+    }
+  };
+
+  const reverseList = () => {
+    setIsReversed(prevIsReversed => !prevIsReversed);
+  };
+
+  const resetList = () => {
+    setIsReversed(false);
+    setSortType(SortType.NONE);
+  };
+
+  const visibleGoods = getReorderedGoods(goodsFromServer, {
+    isReversed,
+    sortType,
+  });
+
+  const alphabetIsActive = sortType === SortType.ALPHABET;
+  const lengthIsActive = sortType === SortType.LENGTH;
```

Comment:
Since you are using both of these in only one place, I don't think it's necessary to create a constant for them